bin/cherry-pick.mjs

```diff
@@ -380,15 +380,17 @@ function reportSummaryNextSteps( successes, failures ) {
 function GHcommentAndRemoveLabel( pr ) {
 	const { number, cherryPickHash } = pr;
 	const comment = prComment( cherryPickHash );
+	const repo = [ '--repo', 'WordPress/gutenberg' ];
```

Comment:
Does it fail because you have multiple origins or something?

Reply:
I have no idea why it's failing for me without explicitly passing it. Could be multiple remotes, yes. Or a different `gh` version? In any case, seems good to pass explicitly?

Reply:
@youknowriad Sorry, I'm completely unfamiliar with `gh`. Looks like it's using a security remote for me, but the `origin` remote is set to this one... 🤷‍♀️ I don't know why it's using that and how to change it.

Reply:
Anyway, we've hardcoded it in other places, so updated those too: https://github.com/WordPress/gutenberg/pull/62643/commits/555040c933d37c3e398f27421b1c6bd6ca3c3a19